src/shop/suppliers/base.py

```diff
@@ -32,3 +32,13 @@ def retrieve_product(self, sku):
         :type sku: str
         :rtype Union[SupplierProduct, None]
         """
+
+    @abstractmethod
+    def order_product(self, sku, qty):
+        """Places an order on product with given SKU.
```

Comment:
What is `SKU`? Please write the full definition at least once before using the abbreviation